pandas/tests/io/formats/data/html/gh47103_expected_output.html

```diff
@@ -0,0 +1,16 @@
+<table border="1" class="dataframe">
```

Comment:
Nit: Could the test that uses this just compare this as a string output?

Reply:
It could, but every test in this test file does it like this, so I just followed convention here. My preference would be to follow a convention, so there is a structure locally, but not a strong opinion.

Reply:
I see. Sure this is fine then